infrastructure/lambda/template.yaml

```diff
@@ -183,6 +207,26 @@ Globals:
           responseLength: $context.responseLength
 
 Mappings:
+  DynatraceLayerARN:
```

Comment:
Now we have the `UseNewDTLayer` condition, do we need this mapping? Perhaps we could just set two constants and select the correct one depending on the state of `UseNewDTLayer`

Reply:
The condition created was to control the creation of the environment variables. As the docs state, the names are changes/some can be removed. 

The mapping defines which ARN should be used in which environment. 

We could use the condition and build an if-else tree at the `Layer` parameter but I think that'll turn out to be more messy.